File: src/kyu_7/jumping_number_special_numbers_series_4/js/solution.js
function jumpingNumber(n){

    let str = String(n);

    for (let i = 0; i < str.length - 1 && str.length > 1; i++) {
        if (Math.abs(str[i] - str[i + 1]) !== 1) {
            return "Not!!";
        }
    }

    return "Jumping!!";
}

module.exports = jumpingNumber;

File: src/kyu_7/jumping_number_special_numbers_series_4/java/Solution.java
package kyu_7.jumping_number_special_numbers_series_4.java;

public class Solution {
    public static String jumpingNumber(int number) {
        if (number <= 9) {
            return "Jumping!!";
        }
        String digits = String.valueOf(number);
        for (int i = 0; i < digits.length(); i++) {
            if (i < digits.length() - 1) {
                if (Math.abs(digits.charAt(i) - digits.charAt(i + 1)) != 1) {
                    return "Not!!";
                }
            }
        }
        return "Jumping!!";
    }
}
